dotCMS/src/main/java/com/dotcms/rendering/velocity/services/PageLoader.java

```diff
@@ -201,9 +201,9 @@ private void addWidgetPreExecuteCodeAndPageInfo(final HTMLPageAsset htmlPage, fi
     public InputStream writeObject(final VelocityResourceKey key) throws DotDataException, DotSecurityException {
 
         HTMLPageAsset page = APILocator.getHTMLPageAssetAPI()
-            .fromContentlet(APILocator.getContentletAPI()
-                .findContentletByIdentifier(key.id1, key.mode.showLive, Long.parseLong(key.language), sysUser(), true));
-
+                .fromContentlet(APILocator.getContentletAPI()
+                .findContentletByIdentifier(key.id1, key.mode.showLive, Long.parseLong(key.language),
```

Comment:
usually is better to use ConversionUtil.toLong(key.language, ()-> ApiLocator.getLanguageAPI().getDefaultLanguage().getId())

Reply:
done https://github.com/dotCMS/core/pull/23022/commits/4d91d0bf79d06f7b4cca64afb251f042ac7520a1#diff-d285e87fa41479227733496517a21cc9f98fbe838f07268bfafec8fc298ddf61R206